assets/js/blocks/active-filters/block.js

```diff
@@ -34,7 +34,7 @@ const ActiveFiltersBlock = ( {
 			return null;
 		}
 		return renderRemovableListItem(
-			__( 'Price:', 'woo-gutenberg-products-block' ),
+			__( 'Price', 'woo-gutenberg-products-block' ),
```

Comment:
There was a double :: here missed in testing.

Reply:
Should we create a PR with this change targeting 2.5.5? That would avoid translators to have to translate this twice.